### PR TITLE
AO3-6829 updates to the dmca policy

### PR DESCRIPTION
**Pull request by Nil-Cipher**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
No changed functionality.
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6829

## Purpose

Update the DMCA page with the new html.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

Go to https://archiveofourown.org/dmca and check for the new content.

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

I have a Jira account but no editing access.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.

*Please note that if you do not fill in this section, we will use your GitHub account name and
they/them pronouns.*

### CODE REVIEW

**Comment by sarken** [comment]
Hi, Nil-Cipher!

Thank you for working on this! Someone will be along to review shortly after you move it out of Draft status. In the meantime, I've updated the Jira issue status to In Review, so no one will mistakenly create a duplicate pull request. 

If you'd like the ability to comment on, assign, and transition issues in the future, you're welcome to create a Jira account! You can just reply here with the account name and we'll set up the permissions for you. (It makes things a bit easier for us on the organizational side if the Full Name on your Jira account either closely matches the name you'd like us to credit in the release notes or includes it in parentheses, e.g. "Nickname (CREDIT NAME).")

Thanks again for contributing! If you have any questions, you can contact us at otw-coders@transformativeworks.org.

**Comment by Nil-Cipher** [comment]
@sarken Do you have an auto-linter? I ran `erblint` and it caught some things, but I don't want to go and manually fix them.

**Comment by Nil-Cipher** [comment]
Found an ERB linter in vscode. This is my first PR, so I didn't touch the i18 stuff. It seemed complicated with the large paragraphs, every `<p>` would need a new variable name and I'm not that creative. 
The test `other_a` is failing because I'm changing the html of the DMCA page.

**Review by sarken — CHANGES_REQUESTED** [review]
I've updated your Jira account permissions and assigned the issue to you! I noticed you haven't filled out the "Credit" section of the pull request template, so I just want to confirm: is Wanda the name you'd like us to credit in the release notes?

I've flagged a few small issues here that need to be corrected, and we'll also need you to update the test that's failing. It should look for other text that appears on this page but not the homepage, like "DMCA takedown notice."

Thanks again!

File: app/views/home/_dmca.html.erb [hunk 1]
```diff
@@ -1,95 +1,109 @@
...
-    <li> <a href="https://www.lumendatabase.org/topics/29#topic-faqs">Lumen DMCA Notices FAQ</a></li>
-    <li> <a href="http://www.newmediarights.org/guide/legal/copyright/dmca/Sample_DMCA_CounterNotice_letter_when_claiming_fair_use_of_a_copyrighted_work">newmediarights.org</a></li>
-  </ol>
-  <p>
-    All notices should be sent to the OTW's designated agent.
-    <hr/>
-    Adapted from Wikipedia's DMCA policy.
-  </p>
-
+   <p>The Digital Millennium Copyright Act (DMCA) establishes the procedure the Organization for Transformative Works (OTW) must follow when users of our sites, including the Archive of Our Own (AO3), <a href="/tos_faq#copyright_plagiarism_faq">reproduce copyrighted material without authorization</a>. Specifically, <a href="https://www.law.cornell.edu/uscode/text/17/512">17 U.S.C. § 512</a> protects the OTW from potential liability when infringing content is present on our sites.</p>
```
[the remark this answers is not in the document]
Comment (Nil-Cipher) [reply app/views/home/_dmca.html.erb:2]: Do you have a recommended linter?

**Comment by Nil-Cipher** [comment]
> I've updated your Jira account permissions and assigned the issue to you! I noticed you haven't filled out the "Credit" section of the pull request template, so I just want to confirm: is Wanda the name you'd like us to credit in the release notes?
> 
> I've flagged a few small issues here that need to be corrected, and we'll also need you to update the test that's failing. It should look for other text that appears on this page but not the homepage, like "DMCA takedown notice."
> 
> Thanks again!

Yeah, nil-cipher (Wanda) is good as the credit name.

**Comment by Nil-Cipher** [comment]
Woops, I tried to squash the commits and messed up the git history, will fix.

**Comment by Nil-Cipher** [comment]
Should be all good now.

File: app/views/home/_dmca.html.erb (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarken) [reply app/views/home/_dmca.html.erb]: Oh, sorry, I thought you said you'd found it! As the [Coding Standards wiki page](https://github.com/otwcode/otwarchive/wiki/Coding-Standards#previewing-reviewdog-comments) mentions, we recommend erblint.

**Review by sarken — APPROVED** [review]
You don't have to change it back since it will be a moot point point when we internationalize the text, but just a heads up that it's a little better to leave the text on one line rather than splitting it across multiple lines to achieve a specific line length. When it's all on one line, fixing a typo/changing one word never requires changes to multiple lines.

Other than that, looks good -- thank you!

File: app/views/home/_dmca.html.erb [hunk 2]
```diff
@@ -1,95 +1,177 @@
...
+  <p>If the account owner believes that their content does not infringe upon your copyright, they may file a DMCA
+    counternotice disputing the takedown. The account owner can file a DMCA counternotice at any point after the
+    content has been removed; there is no time limit.</p>
+  <p>If the OTW receives a valid DMCA counternotice, we will make reasonable efforts to notify you. You will have 10
+    business days to file a lawsuit to prevent the restoration of the content. If you do not notify us within 10
+    business days that you have filed a lawsuit, we will allow the content to be restored in accordance with 17
+    U.S.C. § 512(g)(2)(B)-(C).</p>
+  <h3 class="heading" id="appeal_options">I was notified that my work was removed from AO3 due to a DMCA takedown
+    notice. What can I do?</h3>
+  <p><a href="/content#II.D.">Copyright infringement</a> and <a href="/content#II.E">plagiarism</a> are violations of
```
Comment (lydia-theda) [inline app/views/home/_dmca.html.erb:105]:
```suggestion
  <p><a href="/content#II.D">Copyright infringement</a> and <a href="/content#II.E">plagiarism</a> are violations of
```
Punctuation error in the provided HTML; the live anchor does not end in a period.

**Review by sarken — CHANGES_REQUESTED** [review]
Switching this over to Action Needed, given lydia's comment.

**Review by sarken — APPROVED** [review]

File: app/views/home/_dmca.html.erb [hunk 3]
```diff
@@ -1,95 +1,177 @@
...
+    counternotice disputing the takedown. The account owner can file a DMCA counternotice at any point after the
+    content has been removed; there is no time limit.</p>
+  <p>If the OTW receives a valid DMCA counternotice, we will make reasonable efforts to notify you. You will have 10
+    business days to file a lawsuit to prevent the restoration of the content. If you do not notify us within 10
+    business days that you have filed a lawsuit, we will allow the content to be restored in accordance with 17
+    U.S.C. § 512(g)(2)(B)-(C).</p>
+  <h3 class="heading" id="appeal_options">I was notified that my work was removed from AO3 due to a DMCA takedown
+    notice. What can I do?</h3>
+  <p><a href="/content#II.D">Copyright infringement</a> and <a href="/content#II.E">plagiarism</a> are violations of
+    Sections II.D. and II.E. of our <a href="/tos">Terms of Service</a>, respectively. For more information, please
```
Comment (lydia-theda) [inline app/views/home/_dmca.html.erb:106]: For consistency with the TOS, please remove the trailing periods: 

Sections II.D and II.E of our Terms of Service

Comment (lydia-theda) [inline app/views/home/_dmca.html.erb:106]: requested edit: remove the periods after D and E 

  Sections II.D and II.E

**Pull request merged**